docs/1.0/04-Reference/04-Database/01-Data-Modelling-(SDL).md

```diff
@@ -426,7 +426,7 @@ type Story @model @rename(oldName: "Post") {
 
 The temporary directive `@migrationValue(value: String!)` is used to migrate the value of a scalar field. When changing an optional field to a requried field, it's necessary to also use this directive.
 
-## Naming onventions
+## Naming Conventions
```

Comment:
The rest of the headers use single capital, so `Naming conventions`.

Reply:
Ah, my mistake, thank you for catching that.

Reply:
Fixed.